index.html

```diff
@@ -711,6 +711,43 @@
       </code></pre>
   </div>
 
+  <div class='slider-example'>
```

Comment:
these changes need to be added to the `index.tpl` file (which compile down to this file)